make/testing_flags.mk

```diff
@@ -116,3 +116,6 @@ endif
 
 # Construct the integration test command with the added build flags.
 ITEST_TAGS := $(DEV_TAGS) $(RPC_TAGS) integration $(backend)
+
+# Construct the coverage test command with the added build flags.
+GOACC := $(GOACC_BIN) $(COVER_PKG) -- -tags="$(DEV_TAGS) $(LOG_TAGS)" $(TEST_FLAGS)
```

Comment:
```
-tags="$(DEV_TAGS) $(LOG_TAGS)" $(TEST_FLAGS)
``` 

shouldn't the quote be in the end?

```
-tags="$(DEV_TAGS) $(LOG_TAGS) $(TEST_FLAGS)"
``` 

If it's not a typo, then LGTM :)

Reply:
yeah i think it's needed